### PR TITLE
Feature/add event admin area

### PR DESCRIPTION
**Pull request by FailXSiegen**

### CODE REVIEW

File: src/components/events/event/PendingUsers.vue [hunk 1]
```diff
@@ -1,35 +1,94 @@
...
+      </li>
+    </ul>
+  </div>
 </template>
 
 <script>
 
 import { localize } from '@/helper/localization-helper'
+import { EVENTUSERS_BY_EVENT } from '@/graphql/queries'
+import { updateEventUserVerified } from '@/graphql/mutations'
```
Comment (0m3gaC0d3) [inline src/components/events/event/PendingUsers.vue:26]: Please export the gql as constant like the line above.

File: src/components/events/event/VerifiedUsers.vue [hunk 2]
```diff
@@ -9,27 +9,82 @@
...
+        {{ localize('view.event.user.visitor') }}
+      </button>
     </li>
   </ul>
 </template>
 
 <script>
 import { localize } from '@/helper/localization-helper'
+import { EVENTUSERS_BY_EVENT } from '@/graphql/queries'
+import { updateEventUserVerified } from '@/graphql/mutations'
```
Comment (0m3gaC0d3) [inline src/components/events/event/VerifiedUsers.vue:25]: Same here. Please use a constant.

File: src/components/events/event/VerifiedUsers.vue [hunk 3]
```diff
@@ -9,27 +9,82 @@
...
+        console.error(error)
+      })
     }
   },
-  computed: {
-    verifiedUsers () {
-      return this.$store.getters.verifiedUsers
+  apollo: {
+    eventUsersByEvent: {
+      query: EVENTUSERS_BY_EVENT,
```
Comment (0m3gaC0d3) [inline src/components/events/event/VerifiedUsers.vue:78]: Rename to EVENT**_**USERS_BY_EVENT

File: src/components/events/listing/Row.vue [hunk 4]
```diff
@@ -16,14 +16,15 @@
               :title="localize('view.event.listing.actions.edit')">
         <i class="bi-pencil-square bi--2xl"></i>
       </router-link>
-      <button @click="onInviteLink" class="btn btn-secondary mx-1 my-2"
+      <a :href="'/'+event.slug" class="btn btn-secondary mx-1 my-2"
```
Comment (0m3gaC0d3) [inline src/components/events/listing/Row.vue:19]: Could we use a router link here instead?

File: src/store/index.js [hunk 5]
```diff
@@ -95,6 +45,9 @@ export default new Vuex.Store({
         throw new Error('Could not fetch id from store')
       }
       return id
+    },
+    getCurrentUserVerified: state => {
```
Comment (0m3gaC0d3) [inline src/store/index.js:49]: Rename to isCurrentUserVerfied

File: src/views/Login.vue [hunk 6]
```diff
@@ -63,14 +65,29 @@ export default {
...
     localize (path) {
       return localize(path)
     },
     onLoginById () {
-      alert('Login by ID')
+      fetchEventBySlug(this.eventSlug).then(async (data) => {
+        if (data.success) {
+          await this.$router.push('/' + data.event.slug)
+        } else {
+          const formInputs = document.querySelectorAll('#event-form .form-control')
```
Comment (0m3gaC0d3) [inline src/views/Login.vue:83]: Move to method to reduce duplicated code starting at line 68.

File: test.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+$argon2i$v=19$m=4096,t=3,p=1$XxYNz8vsK4j7R32LhwnX8w$/vGeQr22G6lGk5+YG/VeQDlT6GZLARXEs745LMm8xyI
```
Comment (0m3gaC0d3) [inline test.txt:1]: Remove that file.

File: src/components/events/event/PendingUsers.vue (hunk 1 above)
Comment (FailXSiegen) [reply src/components/events/event/PendingUsers.vue:26]: mutations are defined as functions and not as const

Comment (0m3gaC0d3) [reply src/components/events/event/PendingUsers.vue:26]: nope

Comment (0m3gaC0d3) [reply src/components/events/event/PendingUsers.vue:26]: The old code was also "wrong".

**Review by 0m3gaC0d3 — APPROVED** [review]

**Pull request merged**